include/wx/stdpaths.h

```diff
@@ -257,7 +257,7 @@ class WXDLLIMPEXP_BASE wxStandardPaths : public wxStandardPathsBase
     {
         return m_prefix + wxS("/") + basename;
     }
-
+    virtual wxString GetSharedLibraroesPath() const { return m_prefix; }
```

Comment:
what is this?

Reply:
@MaartenBent ,
It is generic implementation.

Reply:
No, this is an entire different function.

Reply:
@MaartenBent 
> No, this is an entire different function.

Sorry, after looking at the code more close it appears to be a function that will be used if generic verion of the class is used.